tictac.css

```diff
@@ -0,0 +1,58 @@
+body {
+    background-color: #14bdac;
+}
+
+.container {
+    box-sizing: border-box;
+    text-align: center;
+    padding-top: 50px;
+    font-size: large;
```

Comment:
It's better to put font-size for the page on `<html>` tag. That way `rm` will be relative to that.